src/app/header/header.component.ts

```diff
@@ -0,0 +1,37 @@
+import {Component, OnInit} from '@angular/core';
+import {Store} from "@ngrx/store";
+import {Observable} from "rxjs";
+import {selectToggle, selectUserName} from "./state/user/user.selectors";
+import {AppState} from "../content/state/countdown/countdown.state";
+import * as UserActions from "../header/state/user/user.actions"
+import {UserState} from "./state/user/user.state";
+
+@Component({
+  selector: 'app-header',
+  templateUrl: './header.component.html',
+  styleUrls: ['./header.component.css']
+})
+export class HeaderComponent implements OnInit {
+
+  showModal$: Observable<boolean> = this.store.select(selectToggle);
```

Comment:
what has selectToggle to do with showModal ..hmm ? 
for the reader it would be more clear to use more intuitive names .. or just the same wording like: 
```ts
isModalVisible$ = this.store.select(selectIsModalVisible$)
```

Reply:
I use it in html for *ngIf to show/hide ui, ok naming I agree with:)